docs/readme (#1)
highcharts/grid-lite-dist: README.md

# Highcharts Grid Lite

> Highcharts Grid Lite is a lightweight, free version of [Highcharts Grid](https://www.highcharts.com/products/grid), a commercial product for interactive grids from the creators of the [Highcharts](https://github.com/highcharts/highcharts) charting library.

Reviewer: > a commercial product for interactive grids

This sounds a bit weird to me, but I can't think of anything better.

> Highcharts Grid Lite is a lightweight, free version of [Highcharts Grid](https://www.highcharts.com/products/grid), a commercial product for interactive grids from the creators of the [Highcharts](https://github.com/highcharts/highcharts) charting library.

Reviewer 2: ```suggestion
Highcharts Grid Lite is the free version of our [Dashboards Grid](https://www.highcharts.com/docs/dashboards/grid-component) component, from the creators of the [Highcharts](https://github.com/highcharts/highcharts) charting library. Designed to meet modern web standards, Grid Lite is perfect for creating responsive, interactive tables for simple use cases.
```

For the time being we should refer to Dashboards, not the overall Grid product.

> Highcharts Grid Lite is the free version of our [Dashboards Grid](https://www.highcharts.com/docs/dashboards/grid-component) component, from the creators of the [Highcharts](https://github.com/highcharts/highcharts) charting library. Designed to meet modern web standards, Grid Lite is perfect for creating responsive, interactive tables for simple use cases.

Reviewer: ```suggestion
Highcharts Grid Lite is the free, standalone version of our [Dashboards Grid](https://www.highcharts.com/docs/dashboards/grid-component) component, from the creators of the [Highcharts](https://github.com/highcharts/highcharts) charting library. Designed to meet modern web standards optimized for high performance in large datasets, Grid Lite is perfect for creating responsive, interactive tables for simple use cases.
```

- Official website: [highcharts.com](http://www.highcharts.com)
- Product page: [highcharts.com/products/grid](https://www.highcharts.com/products/grid/)
- Documentation: [highcharts.com/docs/grid](https://www.highcharts.com/docs/grid/installation)
- Demos: [highcharts.com/demo](https://www.highcharts.com/demo#highcharts-dashboards-demo-basic)
- Download page: [highcharts.com/download](http://www.highcharts.com/download)
- Licensing: [shop.highcharts.com](https://shop.highcharts.com/)
- Support: [highcharts.com/support](http://www.highcharts.com/support)
- Issues: [github.com/highcharts](https://github.com/highcharts/highcharts/issues)

Highcharts Grid Lite is a [source available](https://en.wikipedia.org/wiki/Source-available_software) product.

> To use full version of Highcharts Grid, you must have a valid license for Highcharts Grid- please refer to [shop.highcharts.com](https://shop.highcharts.com/) for details on licensing.

Reviewer: This does not clearly indicate that this is about Highcharts Grid Pro, may suggest that Grid Lite is paid.

Author: We will include the licence for Lite later on and will be more explicit about it.

> To use full version of Highcharts Grid, you must have a valid license for Highcharts Grid- please refer to [shop.highcharts.com](https://shop.highcharts.com/) for details on licensing.

Reviewer 2: ```suggestion
To use full version of Highcharts Grid, you must have a valid license for Highcharts Dashboards. Please refer to [shop.highcharts.com](https://shop.highcharts.com/) for details on licensing.
```

## Installation

> If you simply want to include Highcharts Grid Lite into a project, use the [distribution package](https://www.npmjs.com/package/@highcharts/grid-lite), or read the [download page](http://www.highcharts.com/download).  

Reviewer 2: ```suggestion
If you simply want to include Highcharts Grid Lite into a project, use the [distribution package](https://www.npmjs.com/package/@highcharts/grid-lite), or refer to the [download page](http://www.highcharts.com/download).  
```

For general installation instructions, see [installation documentation](https://highcharts.com/docs/grid/installation)

### Install from npm
Using npm by running the following command in your terminal:

```Shell
    npm install @highcharts/grid-lite
```

Then import the package in your project:

```JS

>     import Grid from '@highcharts/grid-lite/grid-lite.js'

Reviewer: ```suggestion
    import Grid from '@highcharts/grid-lite/grid-lite.js';
    import '@highcharts/grid-lite/css/grid.css';
```

```

### Use our CDN

Instead of downloading, you can use our CDN to access files directly. See [code.highcharts.com](https://code.highcharts.com) for details.

```HTML

>     <script src="https://code.highcharts.com/grid-lite/grid-lite.js"></script>

Reviewer 2: I know it's not 100% consistent with NPM, but possible to use `https://code.highcharts.com/grid/grid-lite.js` here. So that both versions is in the `https://code.highcharts.com/grid` folder in our CDN?

Reviewer: This will require merging two folder's content into one folder, but I think it should be consistent and the only files that will be overwritten will be the same.

I'll change this in gulptasks and let's treat this as the target path @karolkolodziej.

>     <script src="https://code.highcharts.com/grid-lite/grid-lite.js"></script>

Reviewer: ```suggestion
    <script src="https://code.highcharts.com/grid/grid-lite.js"></script>
```
